data/json/items/vehicle/boat.json

```diff
@@ -15,6 +15,22 @@
     "to_hit": -1,
     "category": "veh_parts"
   },
+  {
+    "type": "GENERIC",
+    "id": "fold_boat_board",
+    "symbol": "o",
+    "color": "brown",
+    "name": { "str": "folding wood boat hull" },
+    "description": "A wooden board that keeps the boat afloat. It can be folded.  Add boat hulls to a vehicle until it floats.  Then attach oars or a motor to get the boat to move.",
```

Comment:
```suggestion
    "description": "A wooden board that keeps the boat afloat.  It can be folded.  Add boat hulls to a vehicle until it floats.  Then attach oars or a motor to get the boat to move.",
```